airflow/configuration.py

```diff
@@ -201,7 +208,9 @@ def _get_cmd_option(self, section, key):
         if (section, key) in self.as_command_stdout:
             if super().has_option(section, fallback_key):
                 command = super().get(section, fallback_key)
-                return run_command(command)
```

Comment:
Why this change? This doesn't look like it would cause a pylint error.

Reply:
Yes, sorry. That was me trying something out, I will revert this change back this weekend.